Spika/Common/Extensions/String+Extension.swift

```diff
@@ -80,3 +80,30 @@ extension String {
         return nil
     }
 }
+
+extension Character {
```

Comment:
Any chance this could have been done better? Perhaps using enums etc.?

Reply:
its used only by string extension, so i dont see need for that